composer.json

```diff
@@ -12,7 +12,7 @@
         "symplify/rule-doc-generator-contracts": "^11.1",
         "nette/utils": "^3.2|^4.0",
         "sebastian/diff": "^5.1|^6.0",
-        "illuminate/container": "^10.39",
+        "illuminate/container": "^10.39|^11.0",
         "webmozart/assert": "^1.11",
         "symfony/yaml": "^6.4",
         "symfony/filesystem": "^6.4"
```

Comment:
Do these need to be updated to support Symfony 7, as Laravel 11 depends uses it? 🤔

Reply:
Well i checked this package on my project which updated to Laravel 11 and both packages have version 7